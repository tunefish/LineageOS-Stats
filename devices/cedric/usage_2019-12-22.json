{
  "1.0.1-cedric": 2,
  "14.1-20170601-unofficial-cedric": 3,
  "14.1-20170706-unofficial-cedric": 19,
  "14.1-20170811-nightly-titan": 1,
  "14.1-20170811-unofficial-cedric": 470,
  "14.1-20171006-unofficial-cedric": 10,
  "14.1-20171018-unofficial-cedric": 15,
  "14.1-20171111_140556-unofficial-cedric": 50,
  "14.1-20180117-unofficial-cedric": 103,
  "14.1-20180204-unofficial-cedric": 1,
  "14.1-20180318-unofficial-cedric": 136,
  "14.1-20180612-unofficial-cedric": 7,
  "14.1-20180618-unofficial-cedric": 1,
  "14.1-20180806-unofficial-cedric": 17,
  "14.1-20180817-unofficial-cedric": 321,
  "14.1-20181103-unofficial-cedric": 1,
  "14.1-20181222-unofficial-cedric": 2,
  "14.1-20190207-unofficial-cedric": 2,
  "14.1-20190221-unofficial-cedric": 3,
  "14.1-20190311-unofficial-cedric": 2,
  "14.1-20190407-unofficial-cedric": 27,
  "14.1-20190410-unofficial-cedric": 1,
  "14.1-20190518-unofficial-cedric": 12,
  "14.1-20190615-unofficial-cedric": 15,
  "14.1-20190619-unofficial-cedric": 7,
  "14.1-20190630-unofficial-cedric": 82,
  "14.1-20190709-unofficial-cedric": 1,
  "14.1-20190710-unofficial-cedric": 15,
  "15.1-20180213-unofficial-cedric": 58,
  "15.1-20180316-unofficial-cedric": 1,
  "15.1-20180415-unofficial-cedric": 398,
  "15.1-20180422-unofficial-cedric": 13,
  "15.1-20180424-unofficial-cedric": 1,
  "15.1-20180611-flo-cedric": 1,
  "15.1-20180715-unofficial-cedric": 1,
  "15.1-20180723-unofficial-cedric": 103,
  "15.1-20180729-unofficial-cedric": 1,
  "15.1-20180809-unofficial-cedric": 1,
  "15.1-20180828-unofficial-cedric": 3,
  "15.1-20180909-unofficial-cedric": 1,
  "15.1-20181007-unofficial-cedric": 1,
  "15.1-20181016-unofficial-cedric": 3,
  "15.1-20181021-unofficial-cedric": 3,
  "15.1-20181216-unofficial-cedric": 1,
  "15.1-20190228-unofficial-cedric": 16,
  "15.1-20190329-unofficial-cedric": 1,
  "15.1-20190407-unofficial-cedric": 1,
  "15.1-20190425-unofficial-cedric": 15,
  "15.1-20190522-unofficial-cedric": 1,
  "15.1-20190608-flo-cedric": 1,
  "15.1-20190615-unofficial-cedric": 26,
  "15.1-20190626-unofficial-cedric": 1,
  "15.1-20190726-unofficial-cedric": 1,
  "16.0-20181108-unofficial-cedric": 2,
  "16.0-20190101-unofficial-cedric": 282,
  "16.0-20190315-unofficial-cedric": 5,
  "16.0-20190405-unofficial-cedric": 368,
  "16.0-20190429-unofficial-cedric": 1,
  "16.0-20190430-unofficial-cedric": 4,
  "16.0-20190501-unofficial-cedric": 1,
  "16.0-20190503-unofficial-cedric": 1,
  "16.0-20190505-unofficial-cedric": 11,
  "16.0-20190507-unofficial-cedric": 2,
  "16.0-20190509-unofficial-cedric": 2,
  "16.0-20190512-unofficial-cedric": 4,
  "16.0-20190514-unofficial-cedric": 1,
  "16.0-20190515-unofficial-cedric": 7,
  "16.0-20190516-unofficial-cedric": 3,
  "16.0-20190518-unofficial-cedric": 2,
  "16.0-20190521-unofficial-cedric": 2,
  "16.0-20190522-unofficial-cedric": 738,
  "16.0-20190525-unofficial-cedric": 49,
  "16.0-20190527-unofficial-cedric": 3,
  "16.0-20190528-unofficial-cedric": 2,
  "16.0-20190529-unofficial-cedric": 3,
  "16.0-20190531-unofficial-cedric": 1,
  "16.0-20190601-unofficial-cedric": 4,
  "16.0-20190602-unofficial-cedric": 12,
  "16.0-20190603-unofficial-cedric": 10,
  "16.0-20190604-unofficial-cedric": 1,
  "16.0-20190605-unofficial-cedric": 1,
  "16.0-20190609-unofficial-cedric": 1,
  "16.0-20190610-unofficial-cedric": 1,
  "16.0-20190623-unofficial-cedric": 2,
  "16.0-20190627-unofficial-cedric": 1,
  "16.0-20190628-unofficial-cedric": 2,
  "16.0-20190701-unofficial-cedric": 57,
  "16.0-20190702-unofficial-cedric": 1,
  "16.0-20190704-unofficial-cedric": 1,
  "16.0-20190707-unofficial-cedric": 4,
  "16.0-20190708-unofficial-cedric": 2,
  "16.0-20190709-unofficial-cedric": 369,
  "16.0-20190711-unofficial-cedric": 32,
  "16.0-20190719-unofficial-cedric": 21,
  "16.0-20190727-unofficial-cedric": 1,
  "16.0-20190729-unofficial-cedric": 1,
  "16.0-20190730-unofficial-cedric": 2,
  "16.0-xtended-20190726-cedric": 14,
  "aosp_cedric-userdebug 9 pq3a.190505.002 eng.zjrdro.20190526.181614 test-keys": 1,
  "aosp_cedric-userdebug 9 pq3a.190505.002 eng.zjrdro.20190527.021046 test-keys": 20,
  "aosp_cedric-userdebug 9 pq3a.190505.002 eng.zjrdro.20190527.160352 test-keys": 1
}