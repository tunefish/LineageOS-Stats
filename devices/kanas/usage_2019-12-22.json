{
  "12.1-20160920-unofficial-sandroidteam-kanas": 1,
  "13.0-20170904-unofficial-kanas-sandroidteam-ih24n-intan": 10,
  "13.0-20181215-unofficial-kanas-sandroidteam-ih24n-intan": 104,
  "13.0-20181216-unofficial-kanas-sandroidteam-ih24n-intan": 8,
  "13.0-20181217-unofficial-kanas-sandroidteam-ih24n-intan": 2,
  "13.0-20181223-unofficial-kanas-sandroidteam-ih24n-intan": 3,
  "13.0-20181226-unofficial-kanas-sandroidteam-ih24n-intan": 51,
  "13.0-20190106-unofficial-homebuilt_tayo-tayo-kanas": 1,
  "14.1-2019/01/20": 1,
  "14.1-20190117-unofficial-kanas-sandroidteam-ih24n-intan": 200,
  "14.1-20190120-unofficial-kanas-sandroidteam-ih24n-intan": 525,
  "14.1-20190722-unofficial-impasta-kanas": 1,
  "15.1-20190213-unofficial-kanas-sandroidteam-ih24n-intan": 136,
  "15.1-20190502-unofficial-kanas-sandroidteam-ih24n": 228,
  "15.1-20190723-unofficial-built_by_tayo-tayo-kanas": 1,
  "kot49h.g355hxxu0apj4": 4
}