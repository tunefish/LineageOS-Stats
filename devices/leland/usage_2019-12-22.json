{
  "15.1-20180422-unofficial-leland": 5,
  "15.1-20180520-unofficial-leland": 1,
  "15.1-20180527-unofficial-leland": 3,
  "15.1-20180619-unofficial-leland": 1,
  "15.1-20180627-unofficial-leland": 2,
  "15.1-20180717-unofficial-leland": 19,
  "15.1-20180904-unofficial-leland": 1,
  "15.1-20181017-unofficial-leland": 1,
  "15.1-20181023-unofficial-leland": 1,
  "15.1-20181102-unofficial-leland": 3,
  "15.1-20181113-unofficial-leland": 15,
  "15.1-20190114-unofficial-leland": 6,
  "15.1-20190117-unofficial-leland": 7,
  "15.1-20190411-unofficial-leland": 62,
  "15.1-20190418-unofficial-leland": 1165
}