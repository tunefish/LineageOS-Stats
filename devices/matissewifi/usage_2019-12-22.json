{
  "14.1-20161117-0709-unofficial-matissewifi": 5,
  "14.1-20170122-unofficial-matissewifi": 58,
  "14.1-20170123-unofficial-matissewifi": 52,
  "14.1-20170130-unofficial-matissewifi": 6,
  "14.1-20170227-unofficial-matissewifi": 5,
  "14.1-20170310-1849-unofficial-matissewifi": 15,
  "14.1-20180107-unofficial-matissewifi": 188,
  "14.1-20180309-unofficial-matissewifi": 12,
  "14.1-20180530-unofficial-matissewifi": 176,
  "14.1-20180812-unofficial-matissewifi": 216,
  "14.1-20181212-unofficial-matissewifi": 11,
  "14.1-20181219-unofficial-matissewifi": 54,
  "14.1-20190410-unofficial-matissewifi": 48,
  "14.1-20190503-unofficial-matissewifi": 71,
  "14.1-20190519-unofficial-matissewifi": 1,
  "14.1-20190519-unofficial-matissewifi-sweetened-xda-build": 47,
  "14.1-20190520-unofficial-matissewifi": 1,
  "14.1-20190529-unofficial-matissewifi": 1,
  "14.1-20190529-unofficial-matissewifi-sweetened-xda-build": 47,
  "14.1-20190609-unofficial-matissewifi-sweetened-personal": 2,
  "14.1-20190620-unofficial-matissewifi-sweetened-xda-build": 91,
  "14.1-20190725-unofficial-matissewifi": 1,
  "14.1-20190729-unofficial-matissewifi": 2
}