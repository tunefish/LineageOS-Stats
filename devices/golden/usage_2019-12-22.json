{
  "12.1": 1,
  "13.0-20170118-unofficial-golden": 1,
  "13.0-20170121-unofficial-golden": 7,
  "13.0-20170129-unofficial-golden": 108,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 17,
  "13.0-20170310-unofficial-golden": 95,
  "13.0-20170328-unofficial-golden": 809,
  "13.0-20170420-unofficial-golden": 18,
  "13.0-20170905-unofficial-glee": 1,
  "14.1-20170122-unofficial-golden": 204,
  "14.1-20170318-unofficial-golden": 6,
  "14.1-20170402-unofficial-golden": 5,
  "14.1-20170415-unofficial-golden": 3,
  "14.1-20170419-unofficial-golden": 10,
  "14.1-20170503-unofficial-golden": 3,
  "14.1-20170507-unofficial-golden": 30,
  "14.1-20170522-unofficial-golden": 5,
  "14.1-20170609-unofficial-golden": 664,
  "14.1-20170610-unofficial-golden": 15,
  "14.1-20170628-unofficial-golden": 869,
  "14.1-20170724-unofficial-golden": 44,
  "14.1-20170817-unofficial-golden": 70,
  "14.1-20170920-unofficial-golden": 16,
  "14.1-20171011-unofficial-golden": 121,
  "14.1-20171215_192311-official-golden": 58,
  "14.1-20171215_192311-unofficial-golden": 1334,
  "14.1-20180109_002536-unofficial-golden": 381,
  "15.1-20181030-unofficial-golden": 8
}